acceptance/Gemfile.lock

```diff
@@ -0,0 +1,112 @@
+GIT
```

Comment:
I'm going back on forth on it. But as @patrick-wright said in the spirit of testing let's not have things locked.